src/router/routes/modules/menu.ts

```diff
@@ -0,0 +1,28 @@
+import { AppRouteRecordRaw } from '@/router/routes/types';
+import { DEFAULT_LAYOUT } from '@/router/routes/base';
+
+const SYSTEM: AppRouteRecordRaw = {
+  path: '/admin',
+  name: 'admin',
```

Comment:
admin? not menu?

Reply:
This should be the root directory, which is the system administration, and the lower level contains the menu administration

Reply:
This should be the root directory, which is the system management, and the lower level contains the menu management

Reply:
This corresponds to the directory in views, I called it system at first, then changed to admin (similar to go-admin)

Reply:
ok. generally system